web-ui/src/pages/PulseReportPage.jsx

```diff
@@ -433,20 +468,22 @@ const PulseReportPage = () => {
   };
 
   const dataInfo = [
-    {key: "internalVeryDissatisfied", stackId: "internal", color: "#273e58", },
-    {key: "internalDissatisfied", stackId: "internal", color: "#1a3c6d", },
-    {key: "internalNeutral", stackId: "internal", color: "#2c519e", },
-    {key: "internalSatisfied", stackId: "internal", color: "#4b7ac7", },
-    {key: "internalVerySatisfied", stackId: "internal", color: "#6fa3e6", },
-    {key: "externalVeryDissatisfied", stackId: "external", color: "#704401", },
-    {key: "externalDissatisfied", stackId: "external", color: "#8a5200", },
-    {key: "externalNeutral", stackId: "external", color: "#b26801", },
-    {key: "externalSatisfied", stackId: "external", color: "#d48a2c", },
-    {key: "externalVerySatisfied", stackId: "external", color: "#e0a456", },
+    {key: "internalVeryDissatisfied", stackId: "internal", color: ociDarkBlue, },
+    {key: "internalDissatisfied", stackId: "internal", color: pSBC(.05, ociDarkBlue), },
+    {key: "internalNeutral", stackId: "internal", color: pSBC(.10, ociDarkBlue), },
+    {key: "internalSatisfied", stackId: "internal", color: pSBC(.15, ociDarkBlue), },
+    {key: "internalVerySatisfied", stackId: "internal", color: pSBC(.2, ociDarkBlue), },
+    {key: "externalVeryDissatisfied", stackId: "external", color: pSBC(-.8, ociOrange), },
+    {key: "externalDissatisfied", stackId: "external", color: pSBC(-.6, ociOrange), },
+    {key: "externalNeutral", stackId: "external", color: pSBC(-.4, ociOrange), },
+    {key: "externalSatisfied", stackId: "external", color: pSBC(-.2, ociOrange), },
+    {key: "externalVerySatisfied", stackId: "external", color: ociOrange, },
```

Comment:
This is to -.9

<img width="1677" alt="Screenshot 2024-12-23 at 9 01 55 AM" src="https://github.com/user-attachments/assets/3764b8f2-1e94-4452-8e21-ce5f91dfe308" />

Reply:
Dark mode...
<img width="1677" alt="Screenshot 2024-12-23 at 9 01 55 AM" src="https://github.com/user-attachments/assets/d20fc5a7-a235-45f4-994c-db303f858b55" />

Reply:
I guess it catches your eye.  The black pie slice looks out of place, to me.

Reply:
Yes.  .9 is better.

Reply:
I had a hard time with the original because the gradient went in diff directions meaning the OCI color meant different things in each case. In one case it was bad, the other good. This breaks my brain less. Not sure it's more pleasing, but it's easier to understand, I think.

Reply:
No, that makes sense!  That definitely crossed my mind.

Reply:
@mkimberlin Are you making that change or do you want me to?

Reply:
Sorry, got distracted. I am about to run out the door to pick up my kid, but I can push my changes and resolve the conflict after I drop her and get into the office.